docs/API_action_commands.md

```diff
@@ -0,0 +1,31 @@
+# Action API supported commands
+
+## Supported actions
+These actions are currently supported using the [Action API](API.md#ActionAPI)
+
+None at this time.
+
+----
+## Actions under development
+These actions are under active development
+
+* deploy site  
+Triggers the initial deployment of a site, using the staged design and secrets
```

Comment:
its possible i am thinking way to ahead right now, but could define what a flow of deploy site is.